pyproject.toml

```diff
@@ -40,3 +42,6 @@ build-backend = "poetry.masonry.api"
 
 [tool.cogex.docker]
 tags = ["cognite/cdf-fabric-replicator:{version}"]
+
+[tool.pytest.ini_options]
+pythonpath = "."
```

Comment:
It could be a me problem but when I run unit test it gives me warnings of `pythonpath` not found as a key. Please verify that this works properly for you or remove this setting.

Reply:
I didn't have a problem running the unit test just now that's weird

Reply:
It's probably some settings on my machine specifically. 

<img width="706" alt="Screenshot 2024-03-15 135215" src="https://github.com/cognitedata/cdf-fabric-replicator/assets/123187956/455e45ea-69c9-48bd-92ae-6fe3cb7a0bd6">